lib/passport-configurator.js

```diff
@@ -294,31 +294,32 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
                     function(err, accessToken) {
                       if (err) {
                         return err.code === 'LOGIN_FAILED'
-                            ? done(null, false, {message: 'Login failed'})
+                            ? done(null, false, {message: errorMsg})
```

Comment:
Basically, it is about not giving out the root of the issue for security reasons, especially in production. We did the same thing last time with the username/password/verification errors. 
The error code is still the same, which can tell the root case. I don't mind reverting back to the main error message. The unit test still catches the error code.